runtime-tests/pom.xml

```diff
@@ -171,6 +171,7 @@
             <wast>func.wast</wast>
             <wast>func_ptrs.wast</wast>
             <wast>global.wast</wast>
+            <wast>i32.wast</wast>
```

Comment:
Is this a regression?

Reply:
not really, just "more correct" checks, expect a bit of forth and back until we have a super solid control flow.